### PR TITLE
Mentions how micropython is run differently

### PR DESCRIPTION
**Pull request by rayluo**
These were the option 2 that I mentioned in [this earlier conversation](https://github.com/ev3dev/ev3dev/issues/1626#issuecomment-2046078842). I hope it can give future `micropython` and `pybricks-micropython` newbies (like me) some useful hint.

### CODE REVIEW

File: docs/programming-languages.md [hunk 1]
```diff
@@ -50,11 +50,24 @@ This brings some advantages, like programs starting significantly faster than
...
 Powered Up smart hubs. Pybricks v2.0 is used by LEGO Education as their [Python
 for EV3](https://education.lego.com/en-us/product-resources/mindstorms-ev3/teacher-resources/python-for-ev3)
 solution and is the same version that is available in ev3dev-stretch.
 
+You may run Pybricks from command line like this:
+`brickrun -r -- pybricks-micropython -c "from pybricks.hubs import EV3Brick; EV3Brick().speaker.beep()"`.
+You may add this as the first line of your Python script,
+`#!/usr/bin/env pybricks-micropython`,
+and then run it from the brick's file browser to run it with Pybricks.
+
```
Comment (dlech) [inline docs/programming-languages.md:70]: Maybe worth mentioning `pybricks-repl` here too?

**Review by dlech — COMMENTED** [review]
Looks like useful info, thanks!

File: docs/programming-languages.md (hunk 1 above)
Comment (rayluo) [reply docs/programming-languages.md:70]: I tried it just now. It seems the `pybricks-repl` is the `pybricks-micropython` in interactive mode, isn't it?

I have been a developer using standard Python for years now, and I hardly find [standard Python's interactive mode](https://docs.python.org/3/tutorial/interpreter.html#interactive-mode) useful. Those `>>>` usually make copy-and-paste miserable, and it is not good for demonstrating a longer code snippet. For one-liners, I would rather use `python -c "print('hello world')"`.

That being said, I can send one more commit mentioning `pybricks-repl`. Stay tuned.

**Comment by dlech** [comment]
Thanks for updating!

**Pull request merged**